.github/workflows/validate.yml

```diff
@@ -11,58 +11,54 @@ jobs:
     strategy:
       matrix:
         BUILD_TYPE: [Release, Debug]
+        DISTRO: ['fedora', 'fedora_rawhide', 'fedora_mpich', 'opensuse', 'ubuntu', 'ubuntu_devel', 'ubuntu_rolling', 'ubuntu_mpich']
+        CXX: ['g++', 'clang++']
+        EXTERNAL_MPI4PY: ['On','Off']
+        include:
+          - BUILD_TYPE: 'Debug'
+            DISTRO: 'fedora_intel'
+            CXX: 'icpc'
+            EXTERNAL_MPI4PY: 'On'
```

Comment:
Why is Intel only available in fedora?

Reply:
Because I only installed it in the Fedora container.

I am sure it is possible under ubuntu as well.